src/components/Header/Header.tsx

```diff
@@ -1,10 +1,42 @@
-import { Button, Flex } from "@adobe/react-spectrum";
+import { ActionButton, Button, Flex, Item, Menu, MenuTrigger } from "@adobe/react-spectrum";
 import { ToggleButton } from "@adobe/react-spectrum";
+import Home from "@spectrum-icons/workflow/Home";
 import Light from "@spectrum-icons/workflow/Light";
 import Moon from "@spectrum-icons/workflow/Moon";
 import { useAuthenticationContext } from "providers/AuthenticationProvider";
 import { useSignatureContext } from "providers/SignatureProvider";
-import React, { FC } from "react";
+import React, { FC, Key } from "react";
+import { useLocation, useNavigate } from "react-router-dom";
+
+const RouteOptions: FC = () => {
+    // destructuring pathname from useLocation hook
+    const { pathname } = useLocation();
```

Comment:
getting the string value from the destructed object though the useLocation hook, getting only the pathname with initial '/'